### ISSUE TITLE
Reading cookies forever

### ISSUE THREAD

**Issue by linck5** [PoE-TradeMacro/POE-TradeMacro#187]
![readingcookies](https://cloud.githubusercontent.com/assets/6393837/21701904/8fe1cfaa-d38f-11e6-957e-54d44e0baee8.png)

I'm having this issue for quite a while. I am left in this screen forever, it never reads the cookies nor does it gives me any error. Even installing version 1.6.0 now, it persists.

**Comment by Eruyome** [comment]
Just curious, does "forever" mean 1 minute+? Maybe even hours+?
If this prevents you from opening the settings menu, open` trade_config.ini` and change
`UseManualCookies =0` to `UseManualCookies =1`, this skips reading those cookies but you'll have to open the settings menu and manually add the cookies there (or simply by editing the .ini).

**Comment by Eruyome** [comment]
poe.trade's cloudflare protection is disabled for now, the current master code skips the whole cookie thing if not needed, will make a release later.

https://github.com/PoE-TradeMacro/POE-TradeMacro/archive/master.zip

**Comment by linck5** [comment]
Thank you very much. Forever means I tested letting it stay for 30 minutes approximately.

Before posting this message, I tested the script again, and it prompted me to download version 1.6.2. I skipped it just to make sure it was still not working with version 1.6.0, and for my surprise it worked. I mean it didn't work, but it showed me the error message that it couldn't read the cookies, so I could at least use the script to check item tears.

But I have to say I've been having this "reading cookies forever" consistently for a long time. I tried to open at least more than 20 times in the past. Of course not letting it sit for 30 minutes each time, but this was a process that lasted 2 to 5 seconds at the most for me usually in the past, when it was working.

Then after this I downloaded version 1.6.2, and now its working perfectly. I can check the prices on poe.trade without any problem. And it doesn't even show the reading cookies message anymore.
